src/test/java/io/jenkins/plugins/sshbuildagents/ssh/ConnectionImplTest.java

```diff
@@ -124,6 +128,7 @@ public void testShellChannel() throws IOException, FormException {
     }
 
     @Test
+    @Disabled("FIXME takes tooooooo long, maybe on CI server only?")
```

Comment:
It is intended because in the first attempt to build this launcher, the connection was closed after 5 minutes. This test will be there until I am confident enough with the launcher. After that, it will be something to launch only manually to test long-running connections.